### PR TITLE
fix(DB/Quest): wrong quest chain order

### PR DESCRIPTION
**Pull request by Helias**
## Changes Proposed:
-  fix the quest chain order
- made the quest `Onu is meditating` available only after taking the quest `The Master's Glaive` that is the first 

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4269

## Tests Performed:
- tested in-game

## How to Test the Changes:
- create a night elf
- `.go cr id 3616`
- Onu should not give the quest `Onu is meditating`
- `.q add 944`
- Onu should give the quest `Onu is meditating`

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1611179465393657785.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611179465393657785');
+UPDATE `quest_template_addon` SET `PrevQuestID` = -944 WHERE `ID`  IN (960, 961);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1611179465393657785.sql:2]: What dows negative prev quest do? Never seen it before

Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1611179465393657785.sql:2]: PrevQuestID

    if value > 0: Contains the previous quest id, that must be completed before this quest can be started.
    If value < 0: Contains the parent quest id, that must be active before this quest can be started.

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Zoidwaffle** [comment]
I believe you are referencing the wrong issue?

https://github.com/azerothcore/azerothcore-wotlk/issues/4269 <-- seems like the proper one

**Comment by Helias** [comment]
Thanks man 😅 , I don't know how it happened

File: data/sql/updates/pending_db_world/rev_1611179465393657785.sql (hunk 1 above)
Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1611179465393657785.sql:2]: me neither, I've just saw the comments that wizzymore posted in the wiki and used it.
I still ahve to make some local test, but I think that it's correct

**Comment by Helias** [comment]
tested in-game and updated the way to test

**Pull request merged**